packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -57,6 +59,13 @@ export default class QuanticSearchBox extends LightningElement {
    * @defaultValue false
    */
   @api textarea = false;
+  /**
+   * Whether to disable rendering the recent queries as suggestions.
+   * @api
+   * @type {boolean}
+   * @defaultValue false
+   */
+  @api disableRecentQuerySuggestions = false;
```

Comment:
We're sure we want it as "default: on" right? Is that what Atomic is doing?

Reply:
Yes in atomic it's by default ON.